nekoyume/Assets/_Scripts/UI/Model/RecipeModel.cs

```diff
@@ -76,7 +76,9 @@ private void LoadEquipment(IEnumerable<EquipmentItemRecipeSheet.Row> recipes)
 
             foreach (var recipe in recipes)
             {
-                var key = GetEquipmentGroup(recipe.ResultEquipmentId);
+                var key = Util.IsEventEquipmentRecipe(recipe.Id)
+                    ? "crystal_equipment"
```

Comment:
If recipe informations are moved to `ScriptableObject`, key should also contained here.